cmd/bridge-device/main.go

```diff
@@ -106,11 +106,17 @@ func patchPropertyElement(td wotTD.ThingDescription, dev *device.Device, endpoin
 }
 
 func getTDOpts(cfg bridgeDevice.Config) ([]device.Option, error) {
-	td, err := bridgeDevice.GetThingDescription(cfg.ThingDescription.File, cfg.NumResourcesPerDevice)
+	// validate TD
+	_, err := bridgeDevice.GetThingDescription(cfg.ThingDescription.File, cfg.NumResourcesPerDevice)
 	if err != nil {
 		return nil, err
 	}
 	return []device.Option{device.WithThingDescription(func(_ context.Context, dev *device.Device, endpoints schema.Endpoints) *wotTD.ThingDescription {
+		// we need to create TD always for concurrency reasons
+		td, err := bridgeDevice.GetThingDescription(cfg.ThingDescription.File, cfg.NumResourcesPerDevice)
```

Comment:
Ideally, we would avoid reading from the file system multiple times. We could read it on line 110 once and then create a deep copy, but I don't see a Clone method on the `ThingDescription` struct. Though we could marshal and unmarshal it to json.